Resolving data files: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 3371/3371 [00:00<00:00, 23864.46it/s]
Found cached dataset imagefolder (/home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f)
Loading cached split indices for dataset at /home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-0b4a8a602117c2d7.arrow and /home/simtoon/.cache/huggingface/datasets/imagefolder/smtn_girls_likeOrNot-8f15de4de8339189/0.0.0/37fbb85cc714a338bea574ac6c7d0b5be5aff46c1862c1989b20e0771199e93f/cache-89c7b4019dbe549b.arrow
Train: 2696 | Test: 675
Traceback (most recent call last):
  File "/home/simtoon/git/attraction_classifier/train.py", line 38, in <module>
    ds["train"]["image"] = [toPNG(image) for image in ds["train"]["image"]]
    ~~~~~~~~~~~^^^^^^^^^
TypeError: 'Dataset' object does not support item assignment